.woodpecker/ci.yml

```diff
@@ -0,0 +1,46 @@
+clone:
+    git:
+        image: woodpeckerci/plugin-git
+        settings:
+            # "partial: false" and depth will copy several commits from the source repo,
+            # enabling history checks for the ocular tool
+            # "lfs: false" disables downloading resources from LFS, which we don't use
+            partial: false
+            depth: 10
+            lfs: false
+
+steps:
+    - name: build
+      when:
+          - event: [ push, pull_request, cron, manual ]
+      image: registry.gitlab.com/fun-tech/fundraising-frontend-docker:latest
+      environment:
+          COMPOSER_CACHE_DIR: /composer_cache
+          TEST_DB_DSN: mysql://fundraising:T3st1ng@database:3306/fundraising
+          GITHUB_TOKEN:
+              from_secret: github_token
+      volumes:
+          - /tmp/woodpeckerci/cache:/composer_cache
+      commands:
+          # Environment setup
+          - echo -e "machine github.com\n  login $GITHUB_TOKEN" > ~/.netrc
+          - composer config -g github-oauth.github.com "$GITHUB_TOKEN"
+          - composer install --prefer-dist --no-progress --no-interaction
+          # CI
+          - vendor/bin/phpcs
+          - php -d memory_limit=1G vendor/bin/phpstan analyse --level=9 --no-progress src/ tests/
+          - vendor/bin/deptrac --no-progress --fail-on-uncovered --report-uncovered
```

Comment:
both PRs are currently failing because deptrac does not seem available, we have to find out why